menu_trans.vim.template

```diff
@@ -1,10 +1,13 @@
-" Menu Translations:    Simplified Chinese
-" Maintainer:           Shun Bai <baishunde@gmail.com>
-" Previous Maintainer:  Yuheng Xie <elephant@linux.net.cn>
-" Last Change:          2022 Feb 19
+" Maintainer:           Ada (Haowen) Yu <me@yuhaowen.com>
+" Previous Maintainer:  Shun Bai <baishunde@gmail.com>
+" Last Change:          2022 Jun 20
 " Original translations
-
+" This template file is created with the tool in
+"
+"       https://github.com/adaext/vim-menutrans-helper
+" The toolbar translation files in Vim will be based on this template file
```

Comment:
No, this comment is not supposed to be talking about the "template" file, but the generated final translations. Your template will be applied to all the existing translations, and this comment will be copied over verbatim. Therefore you should tell the future maintainer that the translation result was generated with your script.